dashboard/app/helpers/levels_helper.rb

```diff
@@ -732,6 +732,12 @@ def blockly_options
     app_options
   end
 
+  def lab2_options
+    app_options = {channel: view_options[:channel], level_id: @level.id}
+    app_options.merge! level_view_options(@level.id)
```

Comment:
do we need all of the `level_view_options`?

Reply:
Technically no at this point; from what I can tell though, this just adds `share`, `iframe_embed`, and `iframe_embed_app_and_code`. Do you think we should just manually pluck out the values we care about? I'd be fine either way

Reply:
I was curious about which properties these were so printed the values. So in terms of readability, I think manually assigning values may be more clear.